HomeWorks/src/main/java/com/epam/spring/homework2/beans/validator/FieldsValidator.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.spring.homework2.beans.validator;
+
+public class FieldsValidator {
+
+    private FieldsValidator() {
```

Comment:
Because you use `private` modifier for constructor -  please think about which modifier is missing at the class declaration

Reply:
final? because in subcalsses can't be called super constructor or am I wrong?

Reply:
Bingo! You got a cookie :) Please add `final` modifier